e2e/scenario_test.go

```diff
@@ -1199,3 +1199,39 @@ func Test_AzureLinuxV2_KubeletCustomConfig(t *testing.T) {
 		},
 	})
 }
+
+func Test_Ubuntu2204ARM64_KubeletCustomConfig(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Tags: Tags{
+			KubeletCustomConfig: true,
+		},
+		Description: "tests that a node on ubuntu 2204 ARM64 bootstrapped with kubelet custom config for seccomp set to non default values",
+		Config: Config{
+			Cluster: ClusterKubenet,
+			VHD:     config.VHDUbuntu2204Gen2Arm64Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				//nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
+				//nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.ContainerService.Properties.OrchestratorProfile.KubernetesConfig.CustomKubeBinaryURL = "https://acs-mirror.azureedge.net/kubernetes/v1.24.9/binaries/kubernetes-node-linux-arm64.tar.gz"
```

Comment:
is this bit necessary to test the code paths you're targeting? I'm just thinking the more we can reduce the amount of places we're hard-coding k8s versions the better

Reply:
agree.. i copied and setup without too much thinking 😅. i should have removed it since the seccomp would not work with 1.24 either 🙏